english/ponder/ponder-about-life.md

```diff
@@ -1,5 +1,4 @@
-You start thinking about how you ended up this bizzare place. Could the marshallmallow wall represent something
-significant? Maybe the lighter represents life, where as fork represents survival. You take another look around 
+You start thinking about how you ended up this bizzare place. Could the marshallmallow wall represent something significant? Maybe the lighter represents life, where as fork represents survival. You take another look around
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.